### ISSUE TITLE
Arena gamemode doesnt work

### ISSUE THREAD

**Issue by coalio** [piqueserver/piqueserver#714]
```
   File "pyspades/player.py", line 1087, in _send_connection_data
      ctf_data.team1_score = blue.score
   File "pyspades/contained.pyx", line 462, in pyspades.contained.CTFState.team1_score.__set__
     unsigned int team1_score, team2_score, cap_limit
 TypeError: an integer is required
```
Very rarely, once in a couple of months this also happens. 
Perhaps the players will connect before completing the initialization. I don’t know what could be more

_Originally posted by @Jipok in https://github.com/piqueserver/piqueserver/issues/589#issuecomment-616532412_

I'm opening an issue because this error still happens. It seems like the original OP fixed it and didn't post the solution.

Everytime I attempt to connect to my server, the error happens:

```
2022-02-10T17:38:38-0600 [stderr#error]     ctf_data.team1_score = blue.score
2022-02-10T17:38:38-0600 [stderr#error]   File "pyspades/contained.pyx", line 462, in pyspades.contained.CTFState.team1_score.__set__
2022-02-10T17:38:38-0600 [stderr#error] TypeError: an integer is required
2022-02-10T17:38:44-0600 [piqueserver.player#info] 127.0.0.1 disconnected
```

I tried this with a compliant arena map, so I am certain it's fault of piqueserver.

If anyone can help, I'll appreciate it

**Comment by 100best** [comment]
> **Stack Trace or logs** [arena.py change map error](https://pastebin.com/tj91nirp)
> 
> I tried commenting this line out and it appears to have fixed it.
> 
> Line 484: `return protocol.on_map_change(self, map)` in `arena.py`

Duplicate of #630

**Comment by utf-4096** [comment]
Should be fixed by #730. If not, feel free to reopen.
